report/2.Analysis/EducationalMiniComputers.tex

```diff
@@ -0,0 +1,30 @@
+\section{Educational mini computers}
+
+Apart from the Arduino board, a lot of other mini computers have been created with educational purposes in mind.
```

Comment:
* Define "mini computer" somewhere in the report. Could be in the list of definitions. Unless it's an actual term in computer science :)

Reply:
Rephrased mini computers to single-board computers, which is an actual term. Also added source about single board pc's which also states that one of the purposes of the SBCs were to be educational. Hope this resolves these comments.